config/clusters/2i2c-aws-us/cluster.yaml

```diff
@@ -1,5 +1,5 @@
 name: 2i2c-aws-us
-provider: aws
+provider: aws # https://2i2c.awsapps.com/start#/
```

Comment:
Perhaps not necessary for this commit but might be worth in the future it might be useful to move the console link from a commento be a defined variable in the cluster.yaml, meaning that it can be extracted programatically (maybe exposed through something like `deployer cluster-console $CLUSTER`) to help oncall flow as the number of consoles and providers grows.

Reply:
@pnasrat good idea, opened #2198 about it!